Sources/DCoreKit/Model/Message.swift

```diff
@@ -104,3 +104,9 @@ public struct MessagePayloadReceiver: Codable {
         nonce
     }
 }
+
+private extension Optional where Wrapped == Address {
+    func isEmptyAddress() -> Bool {
+        return isNil() || self?.publicKey.data.allSatisfy { $0 == 0x00 } ?? true
```

Comment:
this check self?.publicKey.data.allSatisfy { $0 == 0x00 } can be in publickKey as getter

Reply:
done